### ISSUE TITLE
FR - MT Card Row Block (2 Cards Variant)

### ISSUE THREAD

**Issue by DanielaPedrochevd** [Netcentric/vg-macktrucks-com-rd#188]
**Agile Requirement**
As a business owner, I want to display content so that it follows the design, to achieve a more dynamic site: 

- [2 Cards](https://www.figma.com/file/URJhT8pkntqc6h6v5wSLqx/Mack-Core-Design-Kit?type=design&node-id=478-32530&mode=dev)

**Description**

Create a new Card Row block variant (2 cards) that follows the above-mentioned design.  2 Cards is very similar to the Cards block but we would need to add the bold font as well as changing the "Explore" button to a slightly bigger one. 

**Acceptance Criteria**

1. As an editor, I can select the new (2 cards) variant from the Cards row block.
2. The new variant follows the figma design and style mentioned above.

**Comment by DanielaPedrochevd** [comment]
This ticket has to be done after #112

**Comment by shirin27** [comment]
Kamino cloned this issue to [hlxsites/vg-macktrucks-com](https://github.com/hlxsites/vg-macktrucks-com/issues/621)
